frontend/src/components/dialogs/GCreateTerminalSessionDialog.vue

```diff
@@ -238,6 +293,12 @@ export default {
         await this.promptForSelections()
       }
     },
+    defaultTarget (value) {
+      this.targetTab.selectedTarget = value
```

Comment:
default target selection does not work for garden terminal https://localhost:8443/namespace/garden/term/garden#target-tab